### PR TITLE
Utilizing rolify for collection admin and submitters

### PR DESCRIPTION
**Pull request by carolyncole**
Also added migration for UserCollection to roles, so existing collections can have the same access

The plan would be to merge (or deploy to staging) migrate and make sure folks still have access.  After that there would be a follow up PR to delete UserCollection from the system.

### CODE REVIEW

File: app/models/user_collection.rb [hunk 1]
```diff
@@ -20,6 +20,15 @@ def self.add_admin(user_id, collection_id)
     end
   end
 
+  def migrate
+    rolify_fole = if role == "ADMIN"
```
Comment (bess) [inline app/models/user_collection.rb:24]: Should this be `rolify_role`?

File: app/models/user_collection.rb [hunk 2]
```diff
@@ -20,6 +20,15 @@ def self.add_admin(user_id, collection_id)
...
+                    :collection_admin
+                  else
+                    :submitter
+                  end
+    user.add_role rolify_fole, collection
```
Comment (bess) [inline app/models/user_collection.rb:29]: and here

**Review by bess — DISMISSED** [review]
This looks great! I saw one place that I think might be a typo? But I could be wrong. Happy to merge.

File: app/models/user_collection.rb (hunk 1 above)
Comment (carolyncole) [reply app/models/user_collection.rb:24]: Thanks @bess!  That is a typo.  We are not keeping track of Foles 🐴 in the system

**Review by bess — APPROVED** [review]

**Comment by carolyncole** [comment]
refs #332

**Pull request merged**